src/sdk/pynni/nni/compression/torch/pruning/one_shot.py

```diff
@@ -85,12 +85,32 @@ def calc_mask(self, wrapper, wrapper_idx=None):
             return None
 
 class LevelPruner(OneshotPruner):
-    def __init__(self, model, config_list, optimizer=None):
-        super().__init__(model, config_list, pruning_algorithm='level', optimizer=optimizer)
+    """
+    Parameters
+    ----------
+    model : torch.nn.Module
+        Model to be pruned
+    config_list : list
+        Supported keys:
+            - sparsity : This is to specify the sparsity operations to be compressed to.
+            - op_types : Operation types to prune.
+    """
+    def __init__(self, model, config_list):
```

Comment:
`optimizer=None` should not be removed. The example does not work without this parameter.